src/radius/pam/Makefile

```diff
@@ -16,9 +16,14 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	patch -p1 < ../patches/freeradius_configure.patch
 	patch -p1 < ../patches/freeradius_2007-04-06.patch
 	patch -p1 < ../patches/freeradius_libeap_deprecated_openssl_1_0.patch
+	patch -p1 < ../patches/libltdl_config_sub.patch
```

Comment:
Line 20 is overwriting the `config.sub` file, so I think you may want to be patching it after that line (unless the new file is fine, in which case the patch isn't needed?)

I need to do some reading on this, but these two files are autoconf files, and should actually be either updated with `autoreconf -fi` or removed. For now, apply a patch as needed.